<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainWidget</class>
 <widget class="QWidget" name="TerrainWidget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="nameGroupBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Name</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="damageGroupBox">
        <property name="title">
         <string>Damage</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QSpinBox" name="damageSpinBox">
           <property name="minimum">
            <number>-10</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="encounterRateGroupBox">
        <property name="title">
         <string>Encounter rate</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSpinBox" name="encounterRateSpinBox">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="footstepGroupBox">
        <property name="title">
         <string>Footsteps</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="footstepSoundLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="footstepSoundPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="footstepDamageCheckBox">
           <property name="text">
            <string>Play on damage</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="battleHWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="battleBackgroundGroupBox">
        <property name="title">
         <string>Battle background</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QWidget" name="backgroundVWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QWidget" name="backgroundSelectorHWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QRadioButton" name="backgroundBackgroundRadioButton">
                 <property name="text">
                  <string>Background</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="backgroundFrameRadioButton">
                 <property name="text">
                  <string>Frame</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="backgroundView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>320</width>
                <height>240</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>240</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="backgroundSetPushButton">
              <property name="text">
               <string>Set background</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="frameVWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QGroupBox" name="fgFrameGroupBox">
              <property name="title">
               <string>Foreground</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="fgFramePushButton">
                 <property name="text">
                  <string>Set</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="fgFrameHCheckBox">
                 <property name="text">
                  <string>Scroll horizontally</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="fgFrameHSpinBox"/>
               </item>
               <item>
                <widget class="QCheckBox" name="fgFrameVCheckBox">
                 <property name="text">
                  <string>Scroll vertically</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="fgFrameVSpinBox"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="bgFrameGroupBox">
              <property name="title">
               <string>Background</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QPushButton" name="bgFramePushButton">
                 <property name="text">
                  <string>Set</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="bgFrameHCheckBox">
                 <property name="text">
                  <string>Scroll horizontally</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="bgFrameHSpinBox"/>
               </item>
               <item>
                <widget class="QCheckBox" name="bgFrameVCheckBox">
                 <property name="text">
                  <string>Scroll vertically</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="bgFrameVSpinBox"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="oddsDepthVWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="oddsGroupBox">
           <property name="title">
            <string>Battle type odds</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="oddsPreemptiveCheckBox">
              <property name="text">
               <string>Preemptive attack</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="oddsPreemptiveSpinBox">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="oddsBackCheckBox">
              <property name="text">
               <string>Back attack</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="oddsBackSpinBox">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="oddsPincerCheckBox">
              <property name="text">
               <string>Pincer attack</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="oddsPincerSpinBox">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="oddsSurpriseCheckBox">
              <property name="text">
               <string>Surprise attack</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="oddsSurpriseSpinBox">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gridGroupBox">
           <property name="title">
            <string>Grid Depth</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QRadioButton" name="gridShallowRadioButton">
              <property name="text">
               <string>Shallow</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="gridDeepRadioButton">
              <property name="text">
               <string>Deep</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="vehicleSpriteHWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="vehicleGroupBox">
        <property name="title">
         <string>Vehicle interaction</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QCheckBox" name="vehicleShipPassCheckbox">
           <property name="text">
            <string>Ship can pass</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="vehicleBoatPassCheckBox">
           <property name="text">
            <string>Boat can pass</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="vehicleAirshipPassCheckbox">
           <property name="text">
            <string>Airship can pass</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="vehicleAirshipLandCheckBox">
           <property name="text">
            <string>Airship can land</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="spriteGroupBox">
        <property name="title">
         <string>Sprite display</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QRadioButton" name="spriteNormalButton">
           <property name="text">
            <string>Normal</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="spriteThirdRadioButton">
           <property name="text">
            <string>Obscure bottom third</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="spriteHalfRadioButton">
           <property name="text">
            <string>Obscure bottom half</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="spriteSemiRadioButton">
           <property name="text">
            <string>Semi-transparent</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
